interfaces/cython/cantera/test/test_reactor.py

```diff
@@ -1542,3 +1542,39 @@ def test_ConstPressureReactor(self):
         self.assertEqual(states.X[-2], 1)
         for i in range(3,7):
             self.assertNear(states.T[i], states.T[2])
+
+
+class ImplicitSurfChemTest(utilities.CanteraTest):
```

Comment:
I'd suggest a different name for the test suite, rather than dragging in the incomprehensible name of the implementation class which is never seen in user code. Maybe `SurfaceIntegratorTest` or even `AdvanceCoveragesTest`?